server/command.go

```diff
@@ -147,31 +164,147 @@ func (p *Plugin) executeCommandQueue(args *model.CommandArgs) *model.CommandResp
 		message = strings.Join(split[3:], " ")
 	}
 
-	hashtag, error := p.GenerateHashtag(args.ChannelId, nextWeek, weekday)
+	hashtag, error := p.GenerateHashtag(args.ChannelId, nextWeek, weekday, false, time.Now().Weekday())
 	if error != nil {
 		return responsef("Error calculating hashtags. Check the meeting settings for this channel.")
 	}
 
-	searchResults, appErr := p.API.SearchPostsInTeamForUser(args.TeamId, args.UserId, model.SearchParameter{Terms: &hashtag})
+	numQueueItems, itemErr := calculateQueItemNumberAndUpdateOldItems(meeting, args, p, hashtag)
+	if itemErr != nil {
+		return itemErr
+	}
 
+	_, appErr := p.API.CreatePost(&model.Post{
+		UserId:    args.UserId,
+		ChannelId: args.ChannelId,
+		RootId:    args.RootId,
+		Message:   fmt.Sprintf("#### %v %v) %v", hashtag, numQueueItems, message),
+	})
 	if appErr != nil {
-		return responsef("Error calculating list number")
+		return responsef("Error creating post: %s", appErr.Message)
+	}
+
+	return &model.CommandResponse{}
+}
+
+func calculateQueItemNumberAndUpdateOldItems(meeting *Meeting, args *model.CommandArgs, p *Plugin, hashtag string) (int, *model.CommandResponse) {
```

Comment:
Do we need to update old items in this PR?

Reply:
Please follow sequence for reviewing these three PRs  #66->#12->#47. There is one commit for each issue.